en/orm/entities.rst

```diff
@@ -313,7 +313,7 @@ method::
     on.
 
 When using the ``newEntity()`` and ``patchEntity()`` methods in the ``Table``
-objects you also have control over the mass assignment protection. Please refer
+objects you can coustomize mass assignment protection by yourself. Please refer
```

Comment:
coustomize -> customize

but I don't see what was "ambiguous" here... I would leave it as is personally.